CrazyCanvas/Source/RenderStages/PlayerRenderer.cpp

```diff
@@ -476,56 +491,99 @@ namespace LambdaEngine
 			return;
 		}
 
+		// Team members are transparent, Front Culling- and Back Culling is needed
+		RenderCull(false, modFrameIndex, ppFirstExecutionStage, beginRenderPassDesc, viewport, scissorRect, m_PipelineStateIDFrontCull);
+		RenderCull(false, modFrameIndex, ppFirstExecutionStage, beginRenderPassDesc, viewport, scissorRect, m_PipelineStateIDBackCull);
+		// Render enemy with no culling to see backface of paint
+		RenderCull(true, modFrameIndex, ppFirstExecutionStage, beginRenderPassDesc, viewport, scissorRect, m_PipelineStateIDNoCull);
```

Comment:
Might want to render the enemies before the team members. This is because the blending might be wrong if an enemy is in front of a team member.